com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -56,6 +56,7 @@ Using the Inspector allows you to change all of the Decal Projector properties,
 
 - The Decal Projector can affect opaque Materials with either a [Decal Shader](Decal-Shader.md) or a [Decal Master Stack](master-stack-decal.md). However, it can only affect transparent Materials with the [Decal Shader](Decal-Shader.md).
 - Decal Emissive isn't supported on Transparent Material.
+- Decal Emissive is always an additive positive contribution and can't affect the existing emissive properties of the materials assigned to the objects.
```

Comment:
Capitalize "Materials" for standard reasons and, if accurate, change "objects" to "GameObjects":
Materials assigned to GameObjects."